src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/ApplicationTest.kt

```diff
@@ -1307,6 +1307,64 @@ class ApplicationTest : IntegrationTestBase() {
     }
   }
 
+  @Test
+  fun `GET submitted CAS3 application includes assessmentId in the response`() {
```

Comment:
Could this be added to an existing 'generic' integration test? Arguably i'm not sure we need an integration test given we have a unit test too